### PR TITLE
Added bulk alert creation script for NetWeaver provider

### PR DESCRIPTION
**Pull request by adityaku-ms**

### CODE REVIEW

**Review by Adityaj094 — APPROVED** [review]

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1 [hunk 1]
```diff
@@ -0,0 +1,528 @@
+$subscriptionId = ""
+$tenantId = ""
+$rgName = ""
+$amsResourceName = ""
+$alertSuppressionInMinutes = 0
```
Comment (venu-sivanadham) [inline Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1:5]: please make all these variables as script input parameters.

Powershell script input parameters can be defined with rich set of validatoins. Please take a look at [this](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.4#static-parameters)

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1 [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+                }
+                if ($aggregateValue -eq "skip") {
+                    Write-Host "Skipping alert $($alertConfig.name)..."
+                    continue
+                }
+                $query = $query.replace("{Aggregate}", $aggregateValue)
+            }
+            $query = $query.replace("{ProviderInstance}", $providerName)
+            $query = $query.replace("{AlertThreshold}", $alertConfig.alertTemplate.defaultThreshold)
+            $alertName = "[" + $providerName + "] " + $alertConfig.name
```
Comment (venu-sivanadham) [inline Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1:466]: can we check if an alert already exists with this name before create attempt? if it exists, then say that the alert is already created and skipping it.

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+```shell
+git clone https://github.com/Azure/Azure-Monitor-for-SAP-solutions-preview.git
+```
+
+## Prerequisites
+- Either have Azure PowerShell locally installed (https://learn.microsoft.com/powershell/azure/install-azure-powershell), or use Azure Cloud Shell (https://learn.microsoft.com/azure/cloud-shell/overview)
+- Connect to your Azure account using the Connect-AzAccount (https://learn.microsoft.com/powershell/module/az.accounts/connect-azaccount) cmdlet.
+- Make sure you have at least contributor role on the Azure Monitor for SAP solutions resource and the associated managed Resource Group.
+
+## Usage
```
Comment (venu-sivanadham) [inline Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/README.md:14]: I encourage you to add screenshots for these steps, it makes the doc reader friendly.

Also, run the script yourself for a monitor with bunch of NW providers and successfully create the alerts.
Capture screenshots at various stages and put them in the docs as step-by-step process to create the alerts.

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1 (hunk 2 above)
Comment (adityaku-ms) [reply Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1:466]: sure, this is done

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1 (hunk 1 above)
Comment (adityaku-ms) [reply Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/NetWeaverAlertsBulkCreation.ps1:5]: this is done

File: Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/README.md (hunk 3 above)
Comment (adityaku-ms) [reply Scripts/SAP_NetWeaver_Alerts_Bulk_Creation/README.md:14]: done

**Review by venu-sivanadham — APPROVED** [review]

**Review by hirenchalodiya1 — APPROVED** [review]

**Pull request merged**